package/src/mock-builders/generator/channel.js

```diff
@@ -45,6 +45,24 @@ export const generateChannel = (options = { channel: {} }) => {
       },
       frozen: false,
       id,
+      own_capabilities: [
```

Comment:
Not now as it'd block this I think, but would be nice in the future to define these on stream-chat-js, so all js SDKs can get them from there in case of updates to the possible capabilities 

<img src="https://media.giphy.com/media/VFYJXIuuFl6pO/giphy.gif" />

Reply:
Currently it's just typed to `string[]`, but we could have this array exported and use it to type `Channel['own_capabilities']` there I think

Reply:
Yup, going to do that as well!! Just didn't want to block it here

Reply:
This thing was approved and merged by backend people on js client I think
![download](https://user-images.githubusercontent.com/11586388/142733322-1e15a9ea-9f70-4c27-91b8-2725de5c1ce6.jpeg)
!!